fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -168,9 +169,9 @@ const ApuBleed = ({ x, y } : ComponentPositionProps) => {
                 <text
                     x={44}
                     y={48}
-                    className={`FontLarger Right ${adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
+                    className={`FontLarger Right ${apuN.isNormalOperation() && adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
```

Comment:
Up above replace adir1ModeSelector knob with:
```ts
const correctedAverageStaticPressure = useArinc429Var('A32NX_ADIRS_ADR_1_CORRECTED_AVERAGE_STATIC_PRESSURE');
```
Then above `return (`
```ts
// FIXME should be APU bleed absolute pressure label from SDAC
const apuBleedPressAvailable = apuN.isNormalOperation() && correctedAverageStaticPressure.isNormalOperation();
```
Then:
```suggestion
                    className={`FontLarger Right ${apuBleedPressAvailable ? 'Green' : 'Amber'}`}
```